src/main/java/appeng/client/gui/AEBaseGui.java

```diff
@@ -587,7 +579,6 @@ protected boolean checkHotbarKeys( final int keyCode )
 	public void onGuiClosed()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1185)